src/renderer/component/common/form.jsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
```

Comment:
Using `render` and `children` are really similar. Maybe we should just choose one and use it everywhere. The additional feature that `render` gives you is that you can pass state/computed values into the render function and allow consuming components to interact with it in a more customizable way.